sample/mpp-library/src/commonTest/kotlin/ProfileApiTest.kt

```diff
@@ -34,6 +34,80 @@ class ProfileApiTest {
         }
     }
 
+    @Test
+    fun `test return map from api call`() {
+        val testResponse = """
+            {
+                "user1" : {
+                    "name" : "user1"
+                },
+                "user2" : {
+                    "name" : "user2"
+                }
+            }
+        """.trimIndent()
+
+        val httpClient = createMockClient {
+            respondOk(content = testResponse)
+        }
+        val defaultApi = DefaultApi(httpClient = httpClient, json = json)
+
+        val result = runBlocking {
+            defaultApi.apiProfileGetMap()
+        }
+
+        assertTrue { result["user1"]!!.name == "user1" }
```

Comment:
here better will be:
```
assertEquals(
    expected = mapOf("user1" to User(name = "user1"), "user2" to User(name = "user2")),
    actual = result
)
```
but for now - ok

Reply:
Fixed